src/components/Results/PopulationResults.tsx

```diff
@@ -0,0 +1,67 @@
+import { makeStyles, Table, TableRow, TableHead, TableCell, TableBody } from '@material-ui/core';
+import { patientFileState } from '../../state';
+import React from 'react';
+import { PatientHelper } from '../Helpers';
+import { useRecoilValue } from 'recoil';
+
+const fhirpath = require('fhirpath');
+
+const useStyles = makeStyles({
+  table: {
+    width: 100
+  },
+  tableCell: {
+    padding: '0px 10px'
+  }
+});
+
+interface Props {
+  results: any;
```

Comment:
I know I hinted at this before, but this can actually be a stricter type than `any`

(you'll need to import { R4 } from the fhir-types library

```suggestion
  results: R4.IMeasureReport_Group;
```